Explorer/Assets/Scripts/Global/Dynamic/MainSceneLoader.cs

```diff
@@ -199,26 +209,33 @@ private async UniTask InitializeFlowAsync(CancellationToken ct)
                     return;
                 }
 
-
-                Entity playerEntity;
-                (globalWorld, playerEntity) = bootstrap.CreateGlobalWorldAndPlayer(bootstrapContainer, staticContainer!, dynamicWorldContainer!, debugUiRoot);
-
-                dynamicWorldContainer!.InitializeWorldRelatedModules(globalWorld.EcsWorld, playerEntity);
-                staticContainer!.PlayerEntityProxy.SetObject(playerEntity);
+                globalWorld = bootstrap.CreateGlobalWorld(bootstrapContainer, staticContainer!, dynamicWorldContainer!, debugUiRoot, playerEntity);
 
                 await bootstrap.LoadStartingRealmAsync(dynamicWorldContainer!, ct);
 
                 await bootstrap.UserInitializationAsync(dynamicWorldContainer!, globalWorld, playerEntity, splashScreen, ct);
 
-                //TODO: Implement loading this (or more addresses) from Unleash
                 if (debugSettings.portableExperiencesEnsToLoad != null)
                 {
                     foreach (string pxEns in debugSettings.portableExperiencesEnsToLoad)
                     {
-                        await staticContainer.PortableExperiencesController.CreatePortableExperienceByEnsAsync(new ENS(pxEns), ct, true);
+                        staticContainer!.PortableExperiencesController.CreatePortableExperienceByEnsWithErrorHandlingAsync(new ENS(pxEns), ct, true, true).Forget();
+                    }
+                }
+
+                if (debugSettings.EnableRemotePortableExperiences)
+                {
+                    string globalPxFlag = FeatureFlagsConfiguration.GetFlag(FeatureFlags.GLOBAL_PORTABLE_EXPERIENCE);
+
+                    if (staticContainer!.FeatureFlagsCache.Configuration.IsEnabled(globalPxFlag, "csv-variant"))
+                    {
+                        if (!staticContainer.FeatureFlagsCache.Configuration.TryGetCsvPayload(globalPxFlag, "csv-variant", out List<List<string>>? csv)) return;
+
+                        foreach (string value in csv[0]) { staticContainer.PortableExperiencesController!.CreatePortableExperienceByEnsWithErrorHandlingAsync(new ENS(value), ct, true, false).Forget(); }
```

Comment:
well, csv can only be null if TryGetCsvPayload is false, and in that case it will return directly without going further.
In other cases it will return with a non-null csv with at least one element as it will have at least one line (ot it would have returned as false). 
Still the checks wont hurt just in case anything weird should happen xD